### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by nati2323**
- When the user clicks (or taps) the hamburger button, the mobile menu appears.
- When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

**Review by OLIPLICHE — COMMENTED** [review]
Hi @nati2323 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not contain a gitignore file

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+        </button>
+        <button type="button" id="close" >
+          <i class="fas fa-times"></i>
+        </button>
+        <ul class="nav-ul">
+          <li><a href="#portfolio" class="nav-text-1">Portfolio</a></li>
+          <li><a href="#about" class="nav-text-2">About</a></li>
+          <li><a href="#contact" class="nav-text-3">Contact</a></li>
+        </ul>
+      </nav>
```
Comment (codepantha) [inline index.html:38]: - [ ] On your mobile menu, the links except `#portfolio` work when I click them. The reason the portfolio link doesn't work is that you don't have a portfolio section. 

Please what happened to your portfolio section? Kindly fix it.

**Review by codepantha — CHANGES_REQUESTED** [review]
Hi team 👋,

## STATUS: REQUESTED CHANGES ♻️  

Good job so far 👍 !
There are some issues that you still need to work on to go to the next project but you are almost there!



### Highlights ⭐ 🎯 
✔️ Descriptive PR
✔️ Meaningful commit messages
✔️ Linter checks

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
-

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+      <section class="headline" id="portfolio">
```
Comment (nicupop729) [inline index.html]: This section is the headline and not the portfolio per see. 
When the user clicks on the portfolio button, needs to be directed to your work section, which needs to have the projects you built before. 
As we have to respect the Figma design, we need a work section in our portfolio. Kindly, add back the cards with your work history.
This one is the work section that we need to have on the portfolio in accordance with the Figma design chosen by you:
Desktop:
image
Mobile:
image


- [ ] Kindly, implement back on your portfolio the work section with all the projects as in the Figma design and link the "Portfolio" navigation button to this section :+1:

**Review by nicupop729 — CHANGES_REQUESTED** [review]
Hi @nati2323, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
 - [X] Correct GitHub flow practice :heavy_check_mark:
 - [X] Good commit messages :heavy_check_mark:
 - [X] Linters are passing :heavy_check_mark:
 - [X] Clean codebase :heavy_check_mark:


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by nati2323** [comment]
## Work Section
- Updated JS file to add Portfolio section

**Review by Ismailco — APPROVED** [review]
### Hi @nati2323,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**